telemetry/packages/ui/app/components/sidebar/index.tsx

```diff
@@ -37,10 +39,19 @@ export const Sidebar = ({
     [podState, currentPod],
   );
 
+  // Notify when mode of operation is changed
+  useEffect(
+    function notifyModeChange() {
+      log(`Run mode changed to ${currentMode}`);
+    },
+    [currentMode],
+  );
+
   return (
     <main className="col-span-1 h-[100vh] border-l-[0px] border-l-openmct-light-gray px-4 py-8 flex flex-col justify-between bg-hyped-background select-none text-gray-100">
       <div className="flex flex-col gap-12 h-full">
         <PodSelector />
+        <ModeSelector />
```

Comment:
Could we hide this selector behind the `EXTENDED_DEBUG_OPTIONS` environment variable (like the manual state change component in the debug view)? Should be set by the config file and never changed in an "official" run.